compiler/optimizer/OMRRegisterCandidate.cpp

```diff
@@ -2288,7 +2288,7 @@ bool OMR::RegisterCandidates::assign(TR::Block **cfgBlocks, int32_t numberOfBloc
             continue;
         }
 
-        if ((dt.isVector() || dt.isMask()) && !comp()->cg()->hasGlobalVRF()) {
+        if ((dt.isVectorOrMask()) && !comp()->cg()->hasGlobalVRF()) {
```

Comment:
Redundant parentheses.  This line can be changed as `if (dt.isVectorOrMask() && ...) {`.